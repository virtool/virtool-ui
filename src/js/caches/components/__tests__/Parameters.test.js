import CacheParameters from "../Parameters.js";

describe("<CacheParameters />", () => {
    const props = {
        parameters: "foo"
    };

    it("should render", () => {
        const wrapper = shallow(<CacheParameters {...props} />);
        expect(wrapper).toMatchSnapshot();
    });

    it("should rander when [hash=null]", () => {
        props.parameters = null;
        const wrapper = shallow(<CacheParameters {...props} />);
        expect(wrapper).toMatchSnapshot();
    });
});
