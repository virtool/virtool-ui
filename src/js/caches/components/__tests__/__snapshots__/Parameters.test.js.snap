// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CacheParameters /> should rander when [hash=null] 1`] = `
<Box__BoxGroup>
  <Box__BoxGroupHeader>
    Trim Parameters
  </Box__BoxGroupHeader>
  <Table>
    <tbody />
  </Table>
</Box__BoxGroup>
`;

exports[`<CacheParameters /> should render 1`] = `
<Box__BoxGroup>
  <Box__BoxGroupHeader>
    Trim Parameters
  </Box__BoxGroupHeader>
  <Table>
    <tbody>
      <tr
        key="0"
      >
        <Parameters__CacheParameterKey>
          0
        </Parameters__CacheParameterKey>
        <td>
          f
        </td>
      </tr>
      <tr
        key="1"
      >
        <Parameters__CacheParameterKey>
          1
        </Parameters__CacheParameterKey>
        <td>
          o
        </td>
      </tr>
      <tr
        key="2"
      >
        <Parameters__CacheParameterKey>
          2
        </Parameters__CacheParameterKey>
        <td>
          o
        </td>
      </tr>
    </tbody>
  </Table>
</Box__BoxGroup>
`;
